### PR TITLE
Add database migration code for Laravel 5

### PR DESCRIPTION
**Pull request by stobbsm**
Ported the SQL files to Laravel database migration files. Haven't done any seeding yet, but it's a start.

### CODE REVIEW

**Comment by Kovah** [comment]
We don't use the database of v1 for v2.
This is how the db should look like: https://github.com/InvoicePlane/InvoicePlane-V2/wiki/Database

File: config/database.php [hunk 1]
```diff
@@ -26,7 +26,7 @@
 	|
 	*/
 
-	'default' => 'mysql',
```
Comment (Kovah) [inline config/database.php]: Don't use SQLite by default!

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,7 @@
 /node_modules
 /public/assets
 /vendor
-
+.idea/
```
Comment (Kovah) [inline .gitignore]: Please add exclusions like this to the global gitignore file of your computer: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Comment by stobbsm** [comment]
That was a complete brain fart on my part. Will get to work on the new database migrations over next week.

**Comment by Kovah** [comment]
ok but please wait with the user-related migrations until we discussed the problem with the user tables. @fwartner added sentry which brings his own database structure for users and so on.

**Pull request closed without merging**